### DISCUSSION TITLE
differentiate instances

### DISCUSSION THREAD

**Discussion by msjindal** [talmolab/sleap#1589] [Help!]
Is there a way to differentiate instances from one another, for example make one instance always the same rat and the other instance the other rat. I know sleap mixes up the instances when training but is there a way to assign each rat to a certain instance?

**Comment by roomrys** [comment]
Hi @msjindal,

Yep! Select the instance you want to set, then in the menu bar you can set the track with Tracks > Set Instance Track.

Thanks,
Liezl
